proco/connection_statistics/admin.py

```diff
@@ -36,7 +36,7 @@ def has_change_permission(self, request, obj=None):
 
 @admin.register(SchoolWeeklyStatus)
 class SchoolWeeklyStatusAdmin(SchoolNameDisplayAdminMixin, admin.ModelAdmin):
-    list_display = ('get_school_name', 'year', 'week', 'connectivity_status', 'connectivity_type', 'connectivity_speed',
```

Comment:
not priority, but it would be great to show real connectivity/coverage status through admin panel